src/components/CardSelectBottomSheet/CardSelectBottomSheet.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from '@emotion/styled';
+import { CARD_BRANDS } from '@/constant';
+import { Button, Circle, Grid, Typography, VStack } from '@/shared/components';
+import { styleToken } from '@/shared/styles';
+import { CardBrand } from '@/type';
+
+type CardSelectBottomSheetProps = {
+  onOverlayClick?: () => void;
+  onSubmit?: (cardBrand: CardBrand) => void;
+};
+
+export const CardSelectBottomSheet = ({ onOverlayClick, onSubmit }: CardSelectBottomSheetProps) => {
+  const handleOverlayClick = () => {
+    onOverlayClick?.();
+  };
+  return (
+    <BackDrop onClick={handleOverlayClick}>
```

Comment:
```suggestion
  return (
    <BackDrop onClick={onOverlayClick}>
```
이렇게 표현해도 무방하지 않을까요!?

Reply:
> 이렇게 표현해도 무방하지 않을까요!?

불필요한 함수를 하나 더 만들어주고 있었네요 😅